### ISSUE TITLE
WebDriver protocol support

### ISSUE THREAD

**Issue by shamrin** [segmentio/nightmare#792]
**tldr** Any plans to support WebDriver protocol?

We currently screenshot-test our web app the following way:

[Gemini utility](https://github.com/gemini-testing/gemini) ⇔ [PhantomJS](http://phantomjs.org)

Or:

[Gemini utility](https://github.com/gemini-testing/gemini) ⇔ Selenium server ([homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/selenium-server-standalone.rb)) ⇔ [PhantomJS](http://phantomjs.org)

(Arrows "⇔" use [WebDriver protocol](http://www.w3.org/TR/webdriver/).)

It would be very useful to be able to replace PhantomJS (and Selenium?) with Nightmare. Nightmare approach - wrapping Electron - looks very promising. However, I have found no mention of WebDriver protocol support in Nightmare.

Do you plan to support [WebDriver protocol](http://www.w3.org/TR/webdriver/)? Does it fit with Nightmare goals at all?

Probably related:
- https://github.com/nightwatchjs/nightwatch/issues/856
- https://github.com/electron/electron/issues/4834
- Electron seems to provide WebDriver support for [_Electron_ apps](https://github.com/electron/electron/blob/master/docs/tutorial/using-selenium-and-webdriver.md) testing. But I'm looking for support for usual web applications.

**Comment by jessyZu** [comment]
+1

**Comment by vijayanandnandam** [comment]
Webdriver the missing feature for adopting `nightmare` into our rspec feature tests.

**Comment by matthewmueller** [comment]
I don't think this is going to happen on our end, but it would make for a wonderful nightmare or electron wrapper! 

FWIW, here's a minimal server that implements the webdriver protocol: https://github.com/ForbesLindesay/taxi-rank
